lambdas/service/app.py

```diff
@@ -1349,6 +1349,18 @@ def generate_manifest(event, context):
     return {'Location': presigned_url}
 
 
+dss_version_parameter = params.query(
+    'version',
+    schema.optional(str),
+    description='''
+        The version of the file to be checked out. Versions can be
+        lexicographically compared to determine which is more recent.
+        If this parameter is omitted then the most recent version
+        of the file is returned. For more information refer to
+        https://dss.data.humancellatlas.org under `GET /files/{uuid}`.
```

Comment:
```suggestion
        The version of the file to be returned. File versions are opaque strings with only one documented property: they can be
        lexicographically compared with each other in order to determine which version is more recent.
        If this parameter is omitted then the most recent version
        of the file is returned.`.
```

This will soon need to work with TDR, too.